sdk/tables/azure-data-tables/README.md

```diff
@@ -160,12 +133,16 @@ entity = {
     'price': '5'
 }
 ```
-* **Create** - Adds an entity to the table.
-* **Delete** - Deletes an entity from the table.
-* **Update** - Updates an entities information by either merging or replacing the existing entity.
-* **Query** - Queries existing entities in a table based off of the QueryOptions (OData).
-* **Get** - Gets a specific entity from a table by partition and row key.
-* **Upsert** - Merges or replaces an entity in a table, or if the entity does not exist, inserts the entity.
+* **Create** - Add an entity to the table.
```

Comment:
Should these bold operations actually list the function name and link to samples? e.g.
```
**[create_entity](http://link-to-create-sample)** - Adds an entity to the table
```
Or do you prefer to leave it purely conceptual?